### ISSUE TITLE
t/ebug.t fails on some systems

### ISSUE THREAD

**Issue by eserte** [plicease/Devel-ebug#20]
On some of my smoker systems:
```
#   Failed test 'run to end'
#   at /usr/perl5.16.3t/lib/site_perl/5.16.3/Test/Expect.pm line 85.
#          got: 'Program finished. Enter 'restart' or 'q'
# r'
#     expected: 'Program finished. Enter 'restart' or 'q''
# Looks like you failed 1 test of 19.
t/ebug.t ............. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/19 subtests 
```
A quick (and probably incomplete) scan over my test reports revealed that only freebsd systems are affected.

**Comment by plicease** [comment]
I just saw this on Linux in CI.  This is an intermittent failure, because this is the first time I saw it after many good runs, and re-running resulted in a PASS.

**Comment by plicease** [comment]
I think this is resolved in 0.63.
